### PR TITLE
Preset display tweaks

### PR DESCRIPTION
**Pull request by hilburn**
Addded Frost Preset to translations
Added Icons for Shedding, Safety, Manual and Frost Presets

### CODE REVIEW

**Comment by hilburn** [comment]
Very small change, but I thought it would be nice to add icons for the custom presets added by VTherm and to stop "frost" showing in lowercase by adding it to (some of) the translations, and defaulting to "Frost" otherwise

image

File: custom_components/versatile_thermostat/translations/fr.json [hunk 1]
```diff
@@ -542,7 +542,8 @@
             "state": {
               "power": "Délestage",
               "security": "Sécurité",
-              "none": "Manuel"
+              "none": "Manuel",
+              "frost": "Gel"
```
Comment (jmcollin78) [inline custom_components/versatile_thermostat/translations/fr.json]: Hors gel

**Review by jmcollin78 — COMMENTED** [review]
Nice thank you. Just the French translation is "Hors gel"

**Review by jmcollin78 — APPROVED** [review]
Thanks !

**Comment by Shasdo** [comment]
Nice to see that you finally identified it.

Good job !

**Pull request merged**